content/workers/platform/routing/custom-domains.md

```diff
@@ -0,0 +1,40 @@
+---
+pcx-content-type: concept
+title: Custom Domains
+---
+
+# Custom Domains
+
+## About
+Custom Domains allow our customers to connect a Worker to a hostname, without having to fuss with DNS or certificate management.  Cloudflare will create DNS records and issue necessary certificates on behalf of our customers.  The DNS records will point directly to the Worker.
+
+## Building a Custom Domain
+In order to create a Custom Domain, you must have:
+* An active Cloudflare zone on your target domain
+* A Worker to invoke
+
+The interface provides active feedback on valid and invalid entries.  Valid entries are hostnames on an active Cloudflare zone.  If you attempt to create a Custom Domain on a hostname with an existing DNS record, Cloudflare will confirm that you'd like to replace the existing record.  Custom Domains can be attached to your Worker via API or within the Cloudflare dashboard under **Account Home** > [**Workers**](https://dash.cloudflare.com/?zone=workers) > **your Worker** > **Triggers** > **Add Custom Domain**.
```

Comment:
```suggestion
The interface provides active feedback on valid and invalid entries. Valid entries are hostnames on an active Cloudflare zone. If you attempt to create a Custom Domain on a hostname with an existing DNS record, Cloudflare will confirm that you would like to replace the existing record. Custom Domains can be attached to your Worker via API or within the Cloudflare dashboard under **Account Home** > [**Workers**](https://dash.cloudflare.com/?zone=workers) > **your Worker** > **Triggers** > **Add Custom Domain**.
```

By 'the interface' do we mean dashboard? Because if so, we should change it.